deepchem/feat/tests/test_roberta_tokenizer.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from deepchem.feat import RobertaFeaturizer
+
+
+class TestRobertaFeaturizer(unittest.TestCase):
+  """Tests for RobertaFeaturizer"""
+
+  def setUp(self):
+    self.smiles = ["Cn1c(=O)c2c(ncn2C)n(C)c1=O", "CC(=O)N1CN(C(C)=O)C(O)C1O"]
+    self.long_molecule_smiles = [
+        "CCCCCCCCCCCCCCCCCCCC(=O)OCCCNC(=O)c1ccccc1SSc1ccccc1C(=O)NCCCOC(=O)CCCCCCCCCCCCCCCCCCC"
+    ]
+    self.featurizer = RobertaFeaturizer.from_pretrained(
+        "seyonec/SMILES_tokenized_PubChem_shard00_160k")
```

Comment:
Do we have any tokenizers up on the DeepChem HuggingFace? CC @seyonechithrananda

Reply:
Yup! they're included with any models that are uploaded usually - we can change this to a deepchem model for sure